pkg/adapters/grpc/event_grpc.go

```diff
@@ -58,11 +61,25 @@ func (s *EventServiceServer) Publish(ctx context.Context, req *pb.EventPublishRe
 		PayloadType: req.GetEvent().GetPayloadType(),
 		Payload:     req.GetEvent().GetPayload().AsMap(),
 	}
+
+	sp := span.FromHeaders(ctx, "topic-"+req.GetTopic(), map[string][]string{})
+	sp.SetAttributes(
+		semconv.CodeFunctionKey.String("EventService.Publish"),
```

Comment:
Some functions keys are inconsistent:

Here we specify `EventService.<method>`

But in the Queue service we use `Queue.<method>`.